src/main/java/br/com/alura/comex/Processador.java

```diff
@@ -0,0 +1,10 @@
+package br.com.alura.comex;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface Processador {
+	
+	List<Pedido> getPedidos(String caminho_arquivo) throws IOException;
```

Comment:
A convenção do Java é camelCase!

```suggestion
	List<Pedido> getPedidos(String caminhoArquivo) throws IOException;
```